src/store/channels-list/saga.ts

```diff
@@ -92,6 +110,7 @@ export function* fetchConversations() {
     chatClient.getConversations,
   ]);
   yield call(mapToZeroUsers, conversations);
+  yield call(updateOtherMembersLastSeenAt, conversations);
```

Comment:
:thinking: I think this would be better off away from this and just reacting to the channels loaded event that's on the conversationsChannel. (ugg...another use of the word Channel. I think we started using `bus` for these redux-saga ones. haha)

Reply:
Meh, nevermind. Just leave this as is for now.